completion/kubectx.zsh

```diff
@@ -5,8 +5,8 @@ PREV=""
 if [ -f "$KUBECTX" ]; then
     # show '-' only if there's a saved previous context
     local PREV=$(cat "${KUBECTX}")
-    _arguments "1: :((- \
-        $(kubectl config get-contexts --output='name')))"
+    _arguments "1: :($(kubectl config get-contexts --output='name')
+-)"
```

Comment:
we can't do this. `-` has to be the first option.

can you reproduce the problem described in the bug?